src/status_im/data_store/realm/schemas/account/core.cljs

```diff
@@ -1,10 +1,14 @@
 (ns status-im.data-store.realm.schemas.account.core
-  (:require [status-im.data-store.realm.schemas.account.v1.core :as v1]))
+  (:require
+   [status-im.data-store.realm.schemas.account.v1.core :as v1]
+   [status-im.data-store.realm.schemas.account.v2.core :as v2]))
 
 ;; TODO(oskarth): Add failing test if directory vXX exists but isn't in schemas.
 
 ;; put schemas ordered by version
 (def schemas [{:schema        v1/schema
                :schemaVersion 1
-               :migration     v1/migration}])
-
+               :migration     v1/migration}
+              {:schema        v2/schema
```

Comment:
Is it the same migrations as in #4336 or are they going to conflict ?

Reply:
@yenda they are going to conflict, first one that gets in  I will rebase the other